vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/AbstractColumn.java

```diff
@@ -91,6 +96,22 @@ public void setVisible(boolean visible) {
         super.setVisible(visible);
     }
 
+    @Override
+    public void onEnabledStateChanged(boolean enabled) {
+        super.onEnabledStateChanged(enabled);
+
+        // Once https://github.com/vaadin/flow/issues/3998 is fixed, this can be removed.
```

Comment:
This issue seems to be actually fixed.
New issue should be created for ComponentRendered in grid header footer